src/mlpack/methods/ann/activation_functions/softplus_function.hpp

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file softplus_function.hpp
+ * @author Vivek Pal
+ *
+ * Definition and implementation of the softplus function.
+ *
```

Comment:
Yes, sorry I missed that. Booktitle is "Advances in Neural Information Processing Systems".